### PR TITLE
doc: add option to check the bundle against k8s version

### PR DESCRIPTION
**Pull request by camilamacedo86**
Add info over the --optinal-values=k8s=1.22 usage. 

More info:
- https://github.com/operator-framework/operator-sdk/pull/4693
- https://github.com/operator-framework/enhancements/blob/master/enhancements/operator-hub-check.md

### CODE REVIEW

File: content/en/docs/best-practices/common.md [hunk 1]
```diff
@@ -17,6 +17,12 @@ Check and test your operator bundle before you publish it. Note that the [`opera
 operator-sdk bundle validate ./bundle --select-optional suite=operatorframework
 ```
 
+Note that some validators can allow performing the check against the Kubernetes version where that is intend to be published:   
+
+```sh
+operator-sdk bundle validate ./bundle --select-optional suite=operatorframework --optinal-values=k8s=1.22
```
Comment (timflannagan) [inline content/en/docs/best-practices/common.md:23]: I'm not up-to-date on this effort, but should this be `--optional-values`?

```suggestion
operator-sdk bundle validate ./bundle --select-optional suite=operatorframework --optional-values=k8s=1.22
```

**Comment by camilamacedo86** [comment]
/hold

Do not get merged until SDK release 1.7.0

File: content/en/docs/best-practices/common.md [hunk 2]
```diff
@@ -17,6 +17,12 @@ Check and test your operator bundle before you publish it. Note that the [`opera
 operator-sdk bundle validate ./bundle --select-optional suite=operatorframework
 ```
 
+Note that some validators can allow performing the check against the Kubernetes version where that is intend to be published:
```
Comment (estroz) [inline content/en/docs/best-practices/common.md:20]: Lets be specific about this:
```suggestion
The OperatorHub.io validator in the `operatorframework` optional suite allows you to validate that your manifests can work with a Kubernetes cluster of a particular version using the `k8s` optional value:
```

Comment (camilamacedo86) [reply content/en/docs/best-practices/common.md:20]: My intention here is to keep that generic. So, if new validators allow that this doc will not get outdated. 

IHMO the details over what validator and what options are available for each case requires to get done in the API docs https://github.com/operator-framework/api/issues/109. Then, we could link that to other places for further details.

**Review by estroz — APPROVED** [review]
/lgtm

**Comment by camilamacedo86** [comment]
We can move forward here now. 

/hold cancel

**Pull request merged**